src/core/state.go

```diff
@@ -995,9 +995,9 @@ func (state *BuildState) queueTargetAsync(target *BuildTarget, forceBuild, build
 		}
 	}
 	for {
-		called := false
+		var called atomicBool
```

Comment:
I'm surprised -race complained about this one. Can't immediately see where it could go wrong. Unless it tries to be conservative.

Reply:
ye it's not very obvious but the function below that does `Set()` can be called from another goroutine.

This whole thing is benign but it's nicer not to have the complaints - eventually I'd like to be "race clean" on this stuff.